.github/workflows/black.yml

```diff
@@ -9,3 +9,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - uses: psf/black@20.8b1
+        with:
+          options: "--check --diff --color --verbose --line-length 120"
```

Comment:
instead of specifying linting parameters in the github action, what do you think of using the local config file, so that our local runs lint the same way?

We actually did this here: https://github.com/pepkit/pipestat/blob/dev/pyproject.toml

There we're using line length of 99; we shouldn't set this action to 120, I think.